### PR TITLE
Fixed Clientside Crash on dedicated servers when using a LogisticsRequestTable

### PR DESCRIPTION
**Pull request by SonOfTheStars**

### CODE REVIEW

File: common/logisticspipes/pipes/PipeBlockRequestTable.java [hunk 1]
```diff
@@ -472,17 +474,20 @@ protected void onCrafting(ItemStack stack) {
 	}
 
 	public ItemStack getResultForClick() {
-		ItemStack result = getOutput(true);
-		if (result.isEmpty()) {
-			result = getOutput(false);
-		}
-		if (result.isEmpty()) {
+		if(!getWorld().isRemote) {
```
Comment (davboecki) [inline common/logisticspipes/pipes/PipeBlockRequestTable.java]: Please use the method inside the MainProxy to check this.

**Review by davboecki — CHANGES_REQUESTED** [review]
Other than the one thing this looks good.

File: common/logisticspipes/pipes/PipeBlockRequestTable.java [hunk 2]
```diff
@@ -52,6 +52,8 @@
 
 import net.minecraft.util.EnumFacing;
 import net.minecraft.util.text.TextComponentTranslation;
+
+import net.minecraftforge.common.MinecraftForge;
```
Comment (davboecki) [inline common/logisticspipes/pipes/PipeBlockRequestTable.java]: What's this for?

Comment (SonOfTheStars) [reply common/logisticspipes/pipes/PipeBlockRequestTable.java]: needs cleanup

**Review by davboecki — APPROVED** [review]

**Pull request merged**